### ISSUE TITLE
Support motions

### ISSUE THREAD

**Issue by Charlie-83** [ggandor/leap-spooky.nvim#28]
It would be really cool to be able to perform with any motion remotely.

For example `yre` would yank from the point you "leap" to until the end of the word.

**Comment by ggandor** [comment]
I don't think it would make sense to hardcode each possible motion, like text objects, but there is https://github.com/rasulomaroff/telepath.nvim for that (`yr{leap}{arbitrary-motion}`).
